modules/transport/index.js

```diff
@@ -0,0 +1,27 @@
+import { API_ID } from '../API';
+
+import Transport from './Transport';
+
+/**
+ * Option for the default low level transport.
+ *
+ * @type {Object}
+ */
+const defaultTransportOptions = {};
+
+if (typeof API_ID === 'number') {
+    defaultTransportOptions.scope
+        = `jitsi_meet_external_api_${API_ID}`;
+}
+
+export const transport = new Transport({ defaultTransportOptions });
+
+/**
+ * Sets the transport to passed transport.
+ *
+ * @param {Object} newTransport - The new transport.
+ * @returns {void}
+ */
+window.useNewExternalTransport = function(newTransport) {
```

Comment:
Current implementation of the transport module is not related to the API module. The transport module can be used even without the API module existence. 

Also currently the only use case for this function will be the remote control for external apps that doesn't support postMessage and the remote control module is not using the API module.

So I don't think this function fits to APP.API.